packages/suite/src/reducers/suite/suiteReducer.ts

```diff
@@ -460,16 +466,20 @@ export const selectFirmwareRevisionCheckError = (state: StateForFirmwareChecks)
  */
 const selectIsFirmwareRevisionCheckEnabledAndFailed = (state: StateForFirmwareChecks): boolean => {
     const error = selectFirmwareRevisionCheckError(state);
+    const softErrors: FirmwareRevisionCheckError[] = [
+        'cannot-perform-check-offline',
+        'other-error',
+    ];
 
-    return error ? !['cannot-perform-check-offline', 'other-error'].includes(error) : false;
+    return error !== null ? !softErrors.includes(error) : false;
 };
 
 /**
  * Get firmware hash check error, or null if check was successful / skipped.
  */
 export const selectFirmwareHashCheckError = (state: StateForFirmwareChecks) => {
     const device = selectDevice(state);
-    if (!isDeviceAcquired(device) || !device.authenticityChecks) return false;
```

Comment:
This should be `null`, see `selectFirmwareRevisionCheckError`.

I didn't catch it because I didn't do `error !== null` as I should have